event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepository.java

```diff
@@ -198,16 +212,57 @@ protected String jndiName() throws NamingException {
     }
 
     @Override
-    public Stream<Event> getPage(final long offset, final long pageSize,  final Map<String, Object> params) {
+    public boolean recordExists(final long offset, final UUID streamId) {
```

Comment:
is this an offset? or is this now a squenceIdPosition ?

Reply:
it is a position renamed it to a position

Reply:
position